sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/connections/workspace_connection.py

```diff
@@ -176,6 +183,27 @@ def metadata(self, value: Dict[str, Any]):
             return
         self.tags = value
 
+    @property
+    def is_shared(self) -> Dict[str, Any]:
+        """Get the Boolean describing if this connection is shared
+            amongst its cohort within a workspace hub. Only applicable for connections created
+            within a lean workspace.
+        :rtype: bool
+        """
+        return self._is_shared
+
+    @is_shared.setter
+    def is_shared(self, value: bool):
+        """Assign the is_shared property of the connection, determining if it is shared amongst other
+            lean workspaces within its parent workspace hub. Only applicable for connections created
+            within a lean workspace workspace.
+        :param value: The new is_shared value.
+        :type value: bool
+        """
+        if not value:
+            return
+        self._is_shared = value
+
```

Comment:
Why make this a property? 

* It's not read-only since there's a setter
* The setter is just an assignment